archunit/src/main/java/com/tngtech/archunit/core/domain/JavaConstructor.java

```diff
@@ -64,11 +64,22 @@ public boolean isConstructor() {
         return true;
     }
 
+    @Override
+    @PublicAPI(usage = ACCESS)
+    public boolean isMethod() {
+        return false;
```

Comment:
You don't need to override this because the default is `false` anyway, no?

Reply:
Yes, I've missed that.